### PR TITLE
Make the layout friendlier for use with the iPad

### PR DESCRIPTION
**Pull request by rys**
## Description

Remap the layout ever so slightly to be closer to what Apple give you on the Magic Keyboard and iPad Pro Smart Keyboard Folio, putting some of that on the 2nd layer.

Adds a new `config.h` to turn on [grave escape](https://docs.qmk.fm/#/feature_grave_esc?id=grave-escape) support.

Tested using a TADA68 on an iPad Pro!

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tada68/keymaps/mattgemmell/config.h [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#include "../../config.h"
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/mattgemmell/config.h]: This should be changed to:

```suggestion
#pragma once
```

The keyboard's `config.h` is automatically included by the build script, so you don't need to include it here.

File: keyboards/tada68/keymaps/mattgemmell/keymap.c [hunk 2]
```diff
@@ -6,43 +6,43 @@
...
+   * |    |    |    |                       |   |   |    | ← | ↓ | →  |
    * `----------------------------------------------------------------'
    */
   [_FL] = LAYOUT_iso(
-    _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, \
-    _______, KC_BTN1, KC_UP,   KC_BTN2, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
-    _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_HOME, \
-    _______, _______, _______, _______, BL_DEC,  BL_TOGG, BL_INC,  _______, KC_VOLU, KC_VOLD, KC_MUTE, _______, _______, KC_MS_U, KC_END,  \
+    _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, KC_BRMU, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_BRMD, \
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/mattgemmell/keymap.c]: You've got these in the wrong order on this layer, which will lead to keycodes being on the wrong keys. Apply this suggestion and the next to fix it.

```suggestion
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          KC_BRMD, \
```

File: keyboards/tada68/keymaps/mattgemmell/keymap.c [hunk 3]
```diff
@@ -6,43 +6,43 @@
...
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_HOME, \
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/mattgemmell/keymap.c]:
```suggestion
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_HOME, \
```

File: keyboards/tada68/keymaps/mattgemmell/config.h (hunk 1 above)
Comment (rys) [reply keyboards/tada68/keymaps/mattgemmell/config.h]: Resolved in latest commit!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**